doc/examples/reconst_dki.py

```diff
@@ -0,0 +1,314 @@
+"""
+=====================================================================
+Reconstruction of the diffusion signal with the kurtosis tensor model
+=====================================================================
+
+The diffusion kurtois model is an expansion of the diffusion tensor model. In
+addition to the diffusio tensor (DT), the diffusion kurtosis model quantifies
+the diffusion kurtosis tensor (KT) which measures the degree to which water
+diffusion in biologic tissues is non-Gaussian [Jensen2005]_. Measurements of
+non-Gaussian diffusion are of interest since they can be used to charaterize
+tissue microstructural heterogeneity [Jensen2010]_ and to derive concrete
+biophysical parameters as the density of axonal fibres and diffusion tortuosity
+[Fierem2011]_. Moreover, DKI can be used to resolve crossing fibers on
+tractography and to obtain invariant rotational measures not limited to well
+aligned fiber populations [NetoHe2015]_.
+
+The diffusion kurtosis model relates the diffusion-weighted signal,
+$S(\mathbf{n}, b)$, to the applied diffusion weighting, $\mathbf{b}$, the
+signal in the absence of diffusion gradient sensitisation, $S_0$, and the
+values of diffusion, $\mathbf{D(n)}$, and diffusion kurtosis, $\mathbf{K(n)}$,
+along the spatial direction $\mathbf{n}$ [NetoHe2015]_:
+
+.. math::
+    S(n,b)=S_{0}e^{-bD(n)+\frac{1}{6}b^{2}D(n)^{2}K(n)}
+
+$\mathbf{D(n)}$ and $\mathbf{K(n)}$ can be computed from the DT and KT using
+the following equations:
+
+.. math::
+     D(n)=\sum_{i=1}^{3}\sum_{j=1}^{3}n_{i}n_{j}D_{ij}
+
+and
+
+.. math::
+     K(n)=\frac{MD^{2}}{D(n)^{2}}\sum_{i=1}^{3}\sum_{j=1}^{3}\sum_{k=1}^{3}
+     \sum_{l=1}^{3}n_{i}n_{j}n_{k}n_{l}W_{ijkl}
+
+where $D_{ij}$ and $W_{ijkl}$ are the elements of the second-order DT and the
+fourth-order KT tensors, respectively, and $MD$ is the mean diffusivity.
+As the DT, KT has antipodal symmetry and thus only 15 Wijkl elemments are
+needed to fully characterize the KT:
+
+.. math::
+   \begin{matrix} ( & W_{xxxx} & W_{yyyy} & W_{zzzz} & W_{xxxy} & W_{xxxz}
+                    & ... \\
+                    & W_{xyyy} & W_{yyyz} & W_{xzzz} & W_{yzzz} & W_{xxyy}
+                    & ... \\
+                    & W_{xxzz} & W_{yyzz} & W_{xxyz} & W_{xyyz} & W_{xyzz}
+                    & & )\end{matrix}
+
+In the following example we show how to reconstruct your diffusion multi-shell
+datasets using the kurtosis tensor model.
+
+First import the necessary modules:
```

Comment:
There is no need to explain the imports. We did that already in some other tutorials. There is no need to repeat the same information.